spec/bike_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rspec'
+require_relative '../lib/dependencies'
```

Comment:
It should be required in spec/spec_helper.rb

Reply:
Done